ehr/resources/queries/study/demographicsOffspring.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2015-2016 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+SELECT
+    d.id as Id,
+    d2.id  AS Offspring,
+    d2.birth AS date,
+    d2.sire,
```

Comment:
i dont think these fields are accurate at ONPRC? what's the rationale for adding this script to EHR

Reply:
ONPRC has this overridden so it won't affect them. This is added for other PRCs.

Reply:
Yeah ONPRC keeps the dam and sire in the birth dataset and adds a parents node to the animal record in their customizer, so querying dam and sire from demographics is more like d.parents.dam.

Reply:
This is tangential but it's important to be precise on the data. the birth dataset's dam and sire are what's entered at birth (observed data). later, genetic testing can happen (potentially more than one result/record), and those take precedence (hence why demographicsParentage.sql exists). 

One big theme on how EHR is designed is the fact that data changes, even things one might generally assume dont (like your parents).

For here I was musing that maybe there's value in adding calculated SQL fields for demographics.dam and demographics.sire, which might let more core code that assumes those fields exist work. but that's beyond the scope here.